org.eclipse.acute.SWTBotTests/src/org/eclipse/acute/SWTBotTests/dotnetrun/AbstractRunTest.java

```diff
@@ -0,0 +1,48 @@
+package org.eclipse.acute.SWTBotTests.dotnetrun;
```

Comment:
Please put a license header on new files (copy it from an existing file and update it)